### PR TITLE
Tap tooltip to hide

### PR DESCRIPTION
**Pull request by ingridwang**
Addressing #68

### CODE REVIEW

File: lib/src/super_tooltip.dart [hunk 1]
```diff
@@ -43,7 +43,7 @@ class SuperTooltip extends StatefulWidget {
   final double borderWidth;
   final bool? showBarrier;
   final Color? barrierColor;
-  final Rect? touchThrougArea;
+  final Rect? touchThroughArea;
```
Comment (bensonarafat) [inline lib/src/super_tooltip.dart:46]: thanks

File: lib/src/super_tooltip.dart [hunk 2]
```diff
@@ -329,46 +331,52 @@ class _SuperTooltipState extends State<SuperTooltip>
...
-                                BoxShadow(
-                                  blurRadius: shadowBlurRadius,
-                                  spreadRadius: shadowSpreadRadius,
-                                  color: shadowColor,
-                                ),
-                              ]
-                            : null,
-                        shape: BubbleShape(
-                          arrowBaseWidth: widget.arrowBaseWidth,
+                    child: GestureDetector(
```
Comment (bensonarafat) [inline lib/src/super_tooltip.dart:334]: Thanks for the PR
@ingridwang 
Can you make this optional, not everybody will need this feature

**Review by bensonarafat — COMMENTED** [review]
I left a review.
Pls can you work on that before i merge 
Thanks

File: lib/src/super_tooltip.dart (hunk 2 above)
Comment (ingridwang) [reply lib/src/super_tooltip.dart:334]: The field is optional and defaults to false so nothing will happen on tap. Would you prefer I exclude the GestureDetector if the value is false?

File: lib/src/super_tooltip.dart [hunk 3]
```diff
@@ -107,6 +108,7 @@ class SuperTooltip extends StatefulWidget {
     this.borderRadius = 10.0,
     this.overlayDimensions = const EdgeInsets.all(10),
     this.bubbleDimensions = const EdgeInsets.all(10),
+    this.hideTooltipOnTap = false,
```
Comment (bensonarafat) [inline lib/src/super_tooltip.dart:111]: My bad. 
thanks

**Review by bensonarafat — APPROVED** [review]

**Pull request merged**